addons-web-sdk/samples/animation-next-js/src/shared/constants.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Make sure that you modify these constants, if you fork this!
+export const CLOUD_PROJECT_NUMBER: string = "509165100245";
+export const SITE_BASE: string = "https://googleworkspace.github.io/meet/animation-next-js/";
+export const MAIN_STAGE_URL: string = SITE_BASE + "mainstage.html";
+export const SIDE_PANEL_URL: string = SITE_BASE + "sidepanel.html";
+export const ACTIVITY_SIDE_PANEL_URL: string = SITE_BASE + "activitysidepanel.html";
```

Comment:
```suggestion
export const MAIN_STAGE_URL: string = SITE_BASE + "/mainstage.html";
export const SIDE_PANEL_URL: string = SITE_BASE + "/sidepanel.html";
export const ACTIVITY_SIDE_PANEL_URL: string = SITE_BASE + "/activitysidepanel.html";
```

Reply:
also, comments explaining the diff between the three URLs would help readers seeing this for the first time

Reply:
Done. Added jsdoc. Good idea.